File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/AudienceLocation.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class AudienceLocation 
{
    public long Id { get; set; }
    public bool Active { get; set; }
    public bool Valid { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string LocationDescription { get; set; }
    public double Radius { get; set; }
}

File: server-scala-play-server/app/model/AudienceLocation.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for AudienceLocation.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-11T19:31:43.310890579Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class AudienceLocation(
  id: Option[Long],
  active: Option[Boolean],
  valid: Option[Boolean],
  latitude: Option[Double],
  longitude: Option[Double],
  locationDescription: Option[String],
  radius: Option[Double]
)

object AudienceLocation {
  implicit lazy val audienceLocationJsonFormat: Format[AudienceLocation] = Json.format[AudienceLocation]
}

